client/tsconfig.json

```diff
@@ -13,5 +13,8 @@
     "composite": true
   },
   "include": ["src", ".eslintrc.js"],
-  "exclude": ["node_modules", ".vscode-test"]
+  "exclude": ["node_modules", ".vscode-test"],
+	"references": [
+		{ "path": "../common" }

```

Comment:
Other ideas to name the common code: common, shared or lib. I don't know which one is more standard.